package.json

```diff
@@ -53,60 +53,60 @@
-    "@storybook/react": "^5.3.4",
-    "@storybook/theming": "^5.3.4",
-    "@types/jest": "^24.0.22",
-    "@types/lodash.get": "^4.4.6",
-    "@types/lodash.isequal": "^4.5.5",
-    "@types/react": "^16.9.11",
-    "@types/react-redux": "^7.1.5",
-    "@types/react-test-renderer": "^16.9.1",
-    "@types/simonwep__selection-js": "^1.2.0",
-    "@types/storybook-readme": "5.0.4",
-    "@types/styled-components": "5.0.1",
-    "ace-builds": "^1.4.7",
+    "@babel/core": "^7.20.2",
+    "@babel/plugin-transform-runtime": "^7.19.6",
+    "@storybook/addon-options": "^5.3.21",
+    "@storybook/addons": "^5.3.22",
+    "@storybook/react": "^5.3.21",
+    "@storybook/theming": "^6.5.14",
+    "@types/jest": "^29.2.4",
+    "@types/lodash.get": "^4.4.7",
+    "@types/lodash.isequal": "^4.5.6",
+    "@types/react": "^16.14.34",
+    "@types/react-redux": "^7.1.24",
+    "@types/react-test-renderer": "^18.0.0",
+    "@types/simonwep__selection-js": "^1.7.0",
+    "@types/storybook-readme": "^5.0.6",
+    "@types/styled-components": "^5.1.26",
+    "ace-builds": "^1.13.2",
     "awesome-typescript-loader": "^5.2.1",
-    "babel-loader": "^8.0.6",
-    "browserify": "^16.5.0",
-    "commitizen": "^4.0.4",
-    "cz-emoji": "^1.1.2",
-    "gh-pages": "^2.2.0",
-    "jest": "^25.1.0",
+    "babel-loader": "^8.3.0",
+    "browserify": "^17.0.0",
+    "commitizen": "^4.2.6",
+    "cz-emoji": "^1.3.1",
+    "gh-pages": "^4.0.0",
+    "jest": "^25.5.4",
     "lodash.noop": "^3.0.1",
     "markdown-loader": "^5.1.0",
-    "raw-loader": "^3.1.0",
-    "react-ace": "^8.0.0",
+    "raw-loader": "^4.0.2",
+    "react-ace": "^10.1.0",
     "react-docgen-typescript-webpack-plugin": "^1.1.0",
-    "react-markdown": "^4.2.2",
-    "react-split-pane": "^0.1.89",
-    "react-syntax-highlighter": "^11.0.2",
-    "react-test-renderer": "^16.11.0",
-    "rollup": "^1.29.1",
-    "rollup-plugin-babel": "^4.3.3",
+    "react-markdown": "^4.3.1",
+    "react-split-pane": "^0.1.92",
+    "react-syntax-highlighter": "^15.5.0",
+    "react-test-renderer": "^16.14.0",
+    "rollup": "^2.79.1",
+    "rollup-plugin-babel": "^4.4.0",
     "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-replace": "^2.2.0",
-    "rollup-plugin-terser": "^5.2.0",
-    "rollup-plugin-typescript2": "^0.25.3",
-    "source-map-explorer": "^2.1.0",
-    "source-map-loader": "^0.2.4",
-    "storybook-readme": "^5.0.8",
-    "styled-components": "^5.1.0",
-    "terser": "^4.3.9",
-    "ts-jest": "^25.2.1",
+    "rollup-plugin-terser": "^7.0.2",
+    "rollup-plugin-typescript2": "^0.34.1",
+    "source-map-explorer": "^2.5.3",
+    "source-map-loader": "^1.0.0",
+    "storybook-readme": "^5.0.9",
+    "styled-components": "^5.3.6",
+    "terser": "^5.16.1",
+    "ts-jest": "^25.5.1",
     "tslint": "^5.20.1",
     "tslint-config-airbnb": "^5.11.2",
     "tslint-react": "^4.1.0",
-    "typescript": "^3.7.2",
-    "webpack-cli": "^3.3.10"
+    "typescript": "^3.9.10",
```

Comment:
Typescript can't be upgraded because awesome-typescript-loader depends on typescript 2 and 3.